ui/src/App.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import './App.css';
+import LandingPage from './components/LandingPage'
+import '../node_modules/patternfly-react/dist/css/patternfly-react.css'
```

Comment:
@damienomurchu I'm getting an error on this import. 

I did some investigation on the styling [here](https://github.com/laurafitzgerald/tic-tac-toe/pull/2/commits/2f55d5fb6e110134d647e2d8324f82cf5344691e#diff-3253b93f0b46b6279dfc68ee61691a4aR1) which I think you could use here. 
 
Including this line in style.scss and running `npm run build-css` which . you can see in package.json of the linked pr might solve this.

Reply:
Actually hold on that for now. I tried to get it working against this pr and couldn't get the css to build. It is working on the pr that I linked, and when I say working, I mean the button is picking up the styling using the variant property.

Reply:
@laurafitzgerald will re-check it this morning, I swapped out patternfly 4 for patternfly 3, so may be I didn't do so cleanly ;)